app/models/appointment.rb

```diff
@@ -1,5 +1,28 @@
-class Appointment < ApplicationRecord
+class Appointment < ActiveRecord::Base
   validates :name, presence: true
   validates :phone_number, presence: true
   validates :time, presence: true
+
+  after_create :reminder
+
+  # Notify our appointment attendee X minutes before the appointment time
+  def reminder
+    twilio_number = '+14243206946'
+    account_sid = 'AC8b9e42fead05227449ff837386b7e80e'
+    client = Twilio::REST::Client.new account_sid, '59b5bb2b2c8a35a3d46feedfc8803ce6'
+    time_str = ((self.time).localtime).strftime("%I:%M%p on %b. %d, %Y")
+    reminder = "Hi #{self.name}. Just a reminder that you have an appointment coming up at #{time_str}."
+    message = client.api.account.messages.create(
```

Comment:
This is a great start.  Let's talk about handling failure and exceptions!  It's more of an intermediate Ruby topic, but a very important one.

@l1ngoman @jscotty723 @ChristopherJStubbs @kristenmccloud